l10n_br_base/README.rst

```diff
@@ -1,81 +1,77 @@
+    :alt: Beta
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fl10n--brazil-lightgray.png?logo=github
+    :target: https://github.com/OCA/l10n-brazil/tree/10.0-readme-l10n_br_base/l10n_br_base
+    :alt: OCA/l10n-brazil
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/l10n-brazil-10-0-readme-l10n_br_base/l10n-brazil-10-0-readme-l10n_br_base-l10n_br_base
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runbot-Try%20me-875A7B.png
+    :target: https://runbot.odoo-community.org/runbot/124/10.0-readme-l10n_br_base
+    :alt: Try me on Runbot
+
+|badge1| |badge2| |badge3| |badge4| |badge5| 
 
 This module was written to extend the functionality of Company and Partner
 and allow you to fill fields like: CNPJ, IE and etc.
 
-Installation
-============
-
-To install this module, you need to:
-
-* do this ...
-
-Configuration
-=============
-
-To configure this module, you need to:
-
-* go to ...
-
-Usage
-=====
-
-To use this module, you need to:
-
-* go to ...
-
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/{repo_id}/{branch}
+**Table of contents**
 
-
-.. repo_id is available in https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt
-.. branch is "8.0" for example
-
-Known issues / Roadmap
-======================
-
-TODO
+.. contents::
+   :local:
 
 Bug Tracker
 ===========
 
 Bugs are tracked on `GitHub Issues <https://github.com/OCA/l10n-brazil/issues>`_.
 In case of trouble, please check there if your issue has already been reported.
-If you spotted it first, help us smashing it by providing a detailed and welcomed feedback `here <https://github.com/OCA/
-l10n-brazil/issues/new?body=module:%20
-l10n_br_base%0Aversion:%20
-8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/l10n-brazil/issues/new?body=module:%20l10n_br_base%0Aversion:%2010.0-readme-l10n_br_base%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
 
+Do not contact contributors directly about support or help with technical issues.
 
 Credits
 =======
 
+Authors
+~~~~~~~
+
+* Akretion
+
 Contributors
-------------
+~~~~~~~~~~~~
 
 * Renato Lima <renato.lima@akretion.com.br>
 * Raphaël Valyi <raphael.valyi@akretion.com.br>
-* Luis Felipe Mileo <mileo@kmee.com.br> 
-* Michell Stuttgart <michell.stuttgart@kmee.com.br> 
+* Luis Felipe Mileo <mileo@kmee.com.br>
+* Michell Stuttgart <michell.stuttgart@kmee.com.br>
```

Comment:
Essa seção de contributors não deveria sair daqui e ir para CONTRIBUTORS.rst ou a informação fica nos dois lados?

Reply:
This information is already in the CONTRIBUTORS.rst and comes from it